bin/brew

```diff
@@ -21,7 +23,8 @@ then
                          chdir "$BREW_SYMLINK_DIRECTORY" && pwd -P)"
 fi
 
-BREW_LIBRARY_DIRECTORY="$(chdir "$BREW_FILE_DIRECTORY"/../Library && pwd -P)"
+export HOMEBREW_REPOSITORY=$(cd $(dirname $(dirname $HOMEBREW_BREW_FILE)) && pwd -P)
```

Comment:
`HOMEBREW_REPOSITORY` should based on `HOMEBREW_BREW_FILE`'s realpath. So you probably want:
```bash
HOMEBREW_REPOSITORY = "$(chdir "$BREW_FILE_DIRECTORY"/../ && pwd -P)"
```

Also, we seem to prefer to put all of `export` in the very end.